frontends/parsers/v1/v1lexer.ll

```diff
@@ -69,7 +68,7 @@ using Parser = V1::V1Parser;
 
 "@pragma"[ \t]*[A-Za-z_][A-Za-z0-9_]* {
                   BEGIN((driver.saveState = PRAGMA_LINE));
-                  return Parser::make_PRAGMA(StringRef(yytext+7).trim(), driver.yylloc); }
+                  return Parser::make_PRAGMA(cstring(V1Lexer::trim(yytext+7)), driver.yylloc); }
```

Comment:
I am guessing this is needed because you are converting a `string_view` to cstring?Unable to check the interface here right now, maybe the input to `make_PRAGMA` could be string_view itself or just a char pointer?

Reply:
Yes. I've made constructor explicit, so, explicit conversion is needed. `make_PRAGMA` is autogenerated, so changing its interface would require some decent amount of changes here and there. So, maybe next time as a part of https://github.com/p4lang/p4c/issues/4481